### DISCUSSION TITLE
Send Request Block: List of environments limited when re-accessing environments drop down

### DISCUSSION THREAD

**Discussion by askherconsulting** [postmanlabs/postman-flows#328] [Defect]
Link to unlisted youtube video showing the bug in action: https://youtu.be/Lz8IiFkf7ZQ 

**Steps:-**
pre-requisites - there are multiple (in this case 2) environments existing
Add a Send Request block
Add a request
Add an environment e.g. environment A in the environments drop down -> all environments listed plus [none]
Resect the environment drop down e.g. to correct an error 
**Expected**
All selectable environments should be shown
**Actual**
Only 1 environment visible plus [none] is shown

**Impact**
Will be confusing and could prevent user from selecting correct environment.

**Comment by shrey-bhagat** [comment]
This is happening because when you click on Environment dropdown after you have selected one of the environments, the filter is prefilled with the name of the environment. We'll fix this bug ASAP. 

<img width="585" alt="Screenshot 2022-06-08 at 3 31 27 PM" src="https://user-images.githubusercontent.com/98767117/172589742-3966d956-5ce3-413d-86af-746d4ddcac0d.png">


<img width="595" alt="Screenshot 2022-06-08 at 3 31 50 PM" src="https://user-images.githubusercontent.com/98767117/172589825-a9b5b87b-ca09-4685-aead-ad8e0198ac9d.png">

 cc : @saswatds
